install-ci.yml

```diff
@@ -5,10 +5,17 @@
   tags: ['common']
   roles:
     - role: common
+
     - role: dd-agent
       tags:
         - monitoring
 
+    - role: letsencrypt
+      tags:
+        - letsencrypt
```

Comment:
done, though i split it up fairly carefully so it should only get triggered in prod.